lektor/types/formats.py

```diff
@@ -17,3 +23,45 @@ class MarkdownType(Type):
 
     def value_from_raw(self, raw):
         return MarkdownDescriptor(raw.value or u"")
+
+
+# Wrapper with an __html__ method prevents Lektor from escaping HTML tags.
```

Comment:
Is there any reason not to use [`markupsafe.Markup`][1] here (instead of `HTML`)?

[1]: https://markupsafe.palletsprojects.com/en/2.0.x/escaping/